### ISSUE TITLE
Yank 0.10.0 from PyPI once 0.10.1 is available

### ISSUE THREAD

**Issue by hakonanes** [pyxem/orix#395]
As brought to our attention by @sk1p in #394, orix 0.10.0 which only supports Python >= 3.7 is still installable with Python 3.6 due to `python_requires >= 3.7` not being set in the package metadata. This is fixed by him in that PR. @harripj and I agreed (in that PR) that we should make 0.10.0 unavailable from PyPI ([yank](https://pypi.org/help/#yanked) it) once 0.10.1 is available.

**Comment by pc494** [comment]
Just to be a final voice on this, happy with this yank (mainly because it smooths #392) but in general do think yanks are overkill. Would rather not make a habit of it.

**Comment by hakonanes** [comment]
> in general do think yanks are overkill

I agree. But not yanking this time will mean everyone running Python 3.6 will get a broken installation when not specifying a version as long as 0.10.0 is available, effectively hiding orix' support for 3.6 in older releases.

Regarding the conda-forge release of 0.10.0, we don't specify a Python version their either. I tried installing the release in a Python 3.6 conda environment, but conda couldn't solve the environment before filling up my memory... I think we can just add this to the package metadata, increase the build number and release another build of 0.10.0. It should then become unavailable to users of Python 3.6.

**Comment by hakonanes** [comment]
There are now builds of 0.10.0 requiring Python >= 3.7 out on conda-forge, `conda/mamba install orix -c conda-forge` still installs 0.10.0 due to the sole `noarch/orix-0.10.0-pyhd8ed1ab_0` build available still. I've made a PR to patch this here: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/326.

**Comment by hakonanes** [comment]
0.10.0 is now yanked (see [PyPI histori](https://pypi.org/project/orix/#history)). Note that 0.10.0 can still be installed with `pip install orix==0.10.0`, even with Python 3.6. We have the option to un-yank the release (see [PEP 592](https://peps.python.org/pep-0592/#specification)).
